examples/import-sample-export-gnn.ipynb

```diff
@@ -0,0 +1,350 @@
+    "* Sample a part of CORA using the [GDS Random walk with restarts algorithm](https://neo4j.com/docs/graph-data-science/current/algorithms/alpha/rwr/)\n",
+    "* Export the CORA sample client side\n",
+    "* Define and train a Graph Convolutional Neural Network (GCN) on the CORA sample\n",
+    "* Evaluate the GCN on a test set"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c64321df",
+   "metadata": {},
+   "source": [
+    "## Prerequisites\n",
+    "\n",
+    "Running this notebook requires a Neo4j server with a recent GDS version (2.2+) installed.\n",
+    "We recommend using Neo4j Desktop with GDS, or AuraDS.\n",
+    "\n",
+    "Also required are of course the Python libraries:\n",
+    "* `graphdatascience` (see [docs](https://neo4j.com/docs/graph-data-science-client/current/installation/) for installation instructions)\n",
+    "* PyG (see [PyG docs](https://pytorch-geometric.readthedocs.io/en/latest/notes/installation.html) for installation instructions)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3bdf33d6",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "\n",
+    "We start by importing our dependencies and setting up our GDS client connection to the database."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "26fb276b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "from graphdatascience import GraphDataScience\n",
+    "import torch\n",
+    "from torch_geometric.data import Data\n",
+    "import torch.nn.functional as F\n",
+    "from torch_geometric.nn import GCNConv\n",
+    "from torch_geometric.transforms import RandomNodeSplit\n",
+    "import random\n",
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ce14da18",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set seeds for consistent results\n",
+    "random.seed(42)\n",
+    "np.random.seed(42)\n",
+    "torch.manual_seed(42)\n",
+    "torch.cuda.manual_seed_all(42)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "937f1476",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Override NEO4J_URI and NEO4J_AUTH here according to your setup\n",
+    "NEO4J_URI = \"bolt://localhost:7687\"\n",
+    "NEO4J_AUTH = None\n",
+    "gds = GraphDataScience(NEO4J_URI, auth=NEO4J_AUTH)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b1a8ab78",
+   "metadata": {},
+   "source": [
+    "## Sampling CORA\n",
+    "\n",
+    "Next we use the built in CORA loader to get the data into GDS. We will then sample it to get a smaller graph to train on."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a575da60",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "G = gds.graph.load_cora()\n",
+    "\n",
+    "# Let's make sure we have exactly 2708 \"Paper\" nodes\n",
+    "print(G.node_labels())\n",
```

Comment:
I thought about that, but is it really the convention in notebooks? @FlorentinD @nvitucci wdyt?

Reply:
I think asserts are good for testing, but less "interesting" for a user. If the intention is to just show what is happening, I'd just leave the output cell there with the result.

Reply:
I think for notebooks I would prefer the `print`. Dont know about conventions though.

I like the Nicolas approach of removing the comment but leaving the output

Reply:
the advantage of assert is that if this fails it will be easy to see where it failed instead of continuing with silent incorrect data.
I have seen assert in notebooks in MOOCs

Reply:
Agree that asserts could also be nice. And in fact, when we add testing to the notebook that might be the preferred way. But since there are mixed opinions we will leave it as is for now